models/_sector/nft/trades/nft_ethereum_trades_beta.sql

```diff
@@ -24,8 +52,11 @@
         transactions_model=source('ethereum','transactions'),
         tokens_nft_model=ref('tokens_ethereum_nft'),
         tokens_erc20_model=ref('tokens_ethereum_erc20'),
-        prices_model=ref('prices_usd_forward_fill'),
+        prices_model='cte_prices_patch',
         aggregators=ref('nft_ethereum_aggregators'),
         aggregator_markers=ref('nft_ethereum_aggregators_markers')
     )
 }}
+)
+
+select * from enriched_trades
```

Comment:
now both the macro & this model have this select -- is that intentional?

Reply:
yes, it's due to the 2 levels of CTE now present